build.sbt

```diff
@@ -33,7 +33,7 @@ lazy val microservice = Project(appName, file("."))
     dependencyOverrides ++= AppDependencies.overrides,
     retrieveManaged := true,
     evictionWarningOptions in update := EvictionWarningOptions.default.withWarnScalaVersionEviction(warnScalaVersionEviction = false),
-    scalaVersion := "2.12.10"
+    scalaVersion := "2.12.11"
```

Comment:
AFAIK 2.12.13 is the latest - needs checking

Reply:
there is a 2.13.2 now i believe seeing if i can get that to work but not sure

Reply:
ok, uk.gov.hmrc - bootstrap-backend-play-27 only works for 2.12 onwards so the 2.13 one will have to hold off for now